src/widgetastic_patternfly/__init__.py

```diff
@@ -661,14 +661,15 @@ def currently_selected(self):
             for el
             in self.browser.elements(self.CURRENTLY_SELECTED, parent=self)]
 
-    def select(self, *levels, **kwargs):
+    def select(self, *levels, handle_alert=True, **kwargs):
         """Select an item in the navigation.
-
         Args:
             *levels: Items to be clicked in the navigation.
-
         Keywords:
-            anyway: Default behaviour is that if you try selecting an already selected item, it will
+            handle_alert(bool): If set to True, will call self.browser.handle_alert to handle
+            alert popups.
+            anyway: Default behaviour is that if you try selecting an already selected item,
+            it will
```

Comment:
L673 and L672 can fit on the same line I think

Reply:
+1, let's clean this formatting up and then I'm happy to merge.